pkg/kubelet/dockertools/manager.go

```diff
@@ -216,12 +218,30 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 	}
 	dm.runner = lifecycle.NewHandlerRunner(httpClient, dm, dm)
-	if serializeImagePulls {
-		dm.imagePuller = kubecontainer.NewSerializedImagePuller(kubecontainer.FilterEventRecorder(recorder), dm, imageBackOff)
+	dm.containerGC = NewContainerGC(client, containerLogsDir)
+
+	minimumDockerAPIVersionForParallelPullsFlag := false
+	version, err := dm.Version()
+	if err == nil {
+		result, err := version.Compare(minimumDockerAPIVersionForParallelPulls)
```

Comment:
This can probably be extracted into a separate helper func that returns true if Docker supports parallel pulls